reframe/core/pipeline.py

```diff
@@ -1024,9 +1002,24 @@ def check_performance(self):
                 key = '%s:%s' % (self._current_partition.fullname, tag)
                 try:
                     ref, low_thres, high_thres = self.reference[key]
+
+                    perf_extra = {}
+                    if value is not None:
+                        perf_extra['check_perf_value'] = value
+
+                    if  ref is not None:
+                        perf_extra['check_perf_reference'] = ref
+
+                    if low_thres is not None:
+                        perf_extra['check_perf_lower_thres'] = low_thres
+
+                    if high_thres is not None:
+                        perf_extra['check_perf_upper_thres'] = high_thres
```

Comment:
As far as I remember, the Graylog backend needs some arguments passed in the `extra` argument? Could you please remind me what is exactly needed? It seems we need to fine tune a bit this here.